src/backend/expungeservice/parsers/case_parser.py

```diff
@@ -0,0 +1,119 @@
+from html.parser import HTMLParser
+from . import case
+
+
+class CaseParser(HTMLParser):
+    BASE_URI = 'https://publicaccess.courts.oregon.gov/PublicAccessLogin/'
+
+    def __init__(self):
+        HTMLParser.__init__(self)
+        self.cases = []
+        self.column = 0
+        self.within_tr_tag = False
+        self.within_nested_tr = False
+        self.collect_data = False
+        self.case_number = ''      # column 1
+        self.citation_number = []  # column 2
+        self.info = []             # column 3
+        self.date_location = []    # column 4
+        self.type_status = []      # column 5
+        self.charges = []          # column 6+
+        self.case_detail_link = ''
+
+    def handle_starttag(self, tag, attrs):
+        self.__increase_column_count(tag)
+        self.__assign_case_link(tag, attrs)
```

Comment:
Sorry I think I see what your saying. There are no other anchor tags once data collection is turned on, but I think it does make sense to add the extra conditional.

Reply:
Checking for the anchor tag would be good. It would be nice if that check is moved out of the function to here, so it clarifies when we assign case link.